department-of-veterans-affairs/caseflow
# Auto close NoShowHearingTask after 15 days and route correctly

Resolves https://vajira.max.gov/browse/CASEFLOW-212

### Description
- auto close NoShowHearingTask when TimedHoldTask expires
- if appeal is ama, create EvidenceSubmissionWindowTask or if appeal is legacy change location
- remove a test that does not appear to be correct 
- write new tests for the logic

### Testing Plan
1. change disposition of a scheduled hearing to `no_show`
2. run `HearingDispositionChangeJob`
3. go to case details page and ensure you can act upon the no show task without errors
